src/Doctrine/WallabagMigration.php

```diff
@@ -57,4 +62,38 @@ protected function getTable($tableName, $unEscaped = false)
         // return escaped table
         return '`' . $table . '`';
     }
+
+    protected function getForeignKeyName(string $tableName, string $foreignColumnName): string
+    {
+        return $this->generateIdentifierName([$this->getTable($tableName, true), $foreignColumnName], 'fk');
+    }
+
+    protected function getIndexName(string $tableName, $indexedColumnNames): string
+    {
+        $indexedColumnNames = (array) $indexedColumnNames;
+
+        return $this->generateIdentifierName(array_merge([$this->getTable($tableName, true)], $indexedColumnNames), 'idx');
+    }
+
+    protected function getUniqueIndexName(string $tableName, string $indexedColumnName): string
+    {
+        return $this->generateIdentifierName([$this->getTable($tableName, true), $indexedColumnName], 'uniq');
+    }
+
+    /**
+     * @see \Doctrine\DBAL\Schema\AbstractAsset::_generateIdentifierName
+     *
+     * @param string[] $columnNames
+     * @param string   $prefix
+     *
+     * @return string
```

Comment:
Why did you use PHPDoc instead of native typing?

Reply:
I'd say the copy pasting from doctrine code beats me 😅